snapcraft/plugins/python2.py

```diff
@@ -190,34 +191,24 @@ def build(self):
         if not os.path.exists(setup_file):
             return
 
-        os.makedirs(common.get_python2_path(self.installdir), exist_ok=True)
+        os.makedirs(get_python2_path(self.installdir), exist_ok=True)
         self.run(
             ['python2', setup_file,
              'build_ext', '-I{}'.format(_get_python2_include(self.installdir)),
              'install', '--install-layout=deb',
              '--prefix={}/usr'.format(self.installdir),
              ], cwd=self.builddir)
 
+        file_utils.remove_files(self.installdir, lambda x: x.endswith('.pyc'))
+
         # Fix all shebangs to use the in-snap python.
-        common.replace_in_file(self.installdir, re.compile(r''),
-                               re.compile(r'#!.*python'),
-                               r'#!/usr/bin/env python')
+        file_utils.replace_in_file(self.installdir, re.compile(r''),
+                                   re.compile(r'#!.*python'),
+                                   r'#!/usr/bin/env python')
 
     def snap_fileset(self):
         fileset = super().snap_fileset()
         fileset.append('-usr/bin/pip*')
-        fileset.append('-usr/lib/python*/dist-packages/easy-install.pth')
```

Comment:
El 24/08/16 a las 15:08, Leo Arias escribió:
> In snapcraft/plugins/python2.py
> <https://github.com/snapcore/snapcraft/pull/754#discussion_r76108816>:
> 
>>  
>>      def snap_fileset(self):
>>          fileset = super().snap_fileset()
>>          fileset.append('-usr/bin/pip*')
>> -        fileset.append('-usr/lib/python*/dist-packages/easy-install.pth')
> 
> Why is there no need to remove pth anymore?

This is an oversight and we should actually remove all .pth files :-)